packages/plugins/plugin-thread/src/components/ThreadComplementary.tsx

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright 2024 DXOS.org
+//
+import React, { useEffect, useMemo } from 'react';
```

Comment:
newline

Reply:
Should have a lint rule.

Reply:
there actually is a lint rule in the monorepo for the copyright header but it doesn't catch this case haha

Reply:
Maybe we should add the newline to the thing it's searching for.